assets/src/domain/eventEditor/ui/tickets/ticketForm/useTicketFormConfig.tsx

```diff
@@ -34,193 +33,206 @@ const FIELD_NAMES: Array<keyof Ticket> = [
 ];
 
 const useTicketFormConfig = (id: EntityId, config?: EspressoFormProps): TicketFormConfig => {
-	const { startDate: start, endDate: end, ...restProps } = useTicketItem({ id }) || {};
+	const ticket = useTicketItem({ id });
 
 	const { siteTimeToUtc, utcToSiteTime } = useTimeZoneTime();
 
-	const startDate = start ? utcToSiteTime(parseISO(start)) : PLUS_ONE_MONTH;
-	const endDate = end ? utcToSiteTime(parseISO(end)) : PLUS_TWO_MONTHS;
+	const startDate = useMemoStringify(
+		ticket?.startDate ? utcToSiteTime(parseISO(ticket?.startDate)) : setDefaultTime(PLUS_ONE_MONTH, 'start')
+	);
+	const endDate = useMemoStringify(
+		ticket?.endDate ? utcToSiteTime(parseISO(ticket?.endDate)) : setDefaultTime(PLUS_ONE_MONTH, 'end')
+	);
 
 	const { onSubmit } = config;
 
-	const onSubmitFrom: TicketFormConfig['onSubmit'] = ({ dateTime, ...rest }, form, ...restParams) => {
-		// convert "dateTime" object to proper UTC "startDate" and "endDate"
-		const { startDate, endDate } = processDateAndTime(dateTime, siteTimeToUtc);
+	const onSubmitFrom: TicketFormConfig['onSubmit'] = useCallback(
+		({ dateTime, ...rest }, form, ...restParams) => {
+			// convert "dateTime" object to proper UTC "startDate" and "endDate"
+			const { startDate, endDate } = processDateAndTime(dateTime, siteTimeToUtc);
 
-		const values = { ...rest, startDate, endDate };
+			const values = { ...rest, startDate, endDate };
 
-		return onSubmit(values, form, ...restParams);
-	};
+			return onSubmit(values, form, ...restParams);
+		},
+		[onSubmit, siteTimeToUtc]
```

Comment:
do we need to include `processDateAndTime` ?

Reply:
No, that's imported from another module and doesn't change between renders